src/features/attributeSelect.tsx

```diff
@@ -567,16 +630,99 @@ const TabsPanel = ({
-              intent="primary"
-              text={"Find All Current Values"}
-              rightIcon={"search"}
-              onClick={() => {
-                const potentialOptions = findAllPotentialOptions(attributeName);
-                setPotentialOptions(potentialOptions);
-                setShowPotentialOptions(true);
-              }}
-            />
+            <>
+              <FormGroup 
+                label={
+                  <div className="flex items-center gap-2">
+                    <span>Display Format</span>
+                    <Tooltip
+                      content="JavaScript function that takes 'text' as input and returns formatted string"
+                      placement="top"
+                    >
+                      <Icon icon="info-sign" className="opacity-80" />
+                    </Tooltip>
+                  </div>
+                } 
+                className="m-0"
+              >
+                <div className="flex flex-col space-y-2">
+                  <div className="flex items-center gap-2">
+                    <MenuItemSelect
+                      items={Object.keys(TEMPLATE_MAP)}
+                      onItemSelect={(template) => {
+                        const newFunction = TEMPLATE_MAP[template as TemplateName];
+                        setSelectedTemplate(template);
+                        setTransformFunction(newFunction);
+                        setInputSetting({
+                          blockUid: attributeUid,
+                          key: "format",
+                          value: newFunction,
+                        });
+                        setInputSetting({
+                          blockUid: attributeUid,
+                          key: "template",
+                          value: template,
+                        });
+                      }}
+                      activeItem={selectedTemplate}
+                    />
+                    <Tooltip  
+                      content={
+                        <div className="text-sm">
+                          <p className="font-bold mb-2">Available Templates:</p>
+                          <ul className="list-disc list-inside space-y-1">
+                            {Object.entries(TEMPLATE_MAP).map(([name, func]) => (
+                              <li key={name}>
+                                <span className="font-mono">{name}:</span>{" "}
+                                {name === "Remove Double Brackets" && "Removes [[text]] format"}
+                                {name === "Convert to Uppercase" && "Makes text all caps"}
+                                {name === "Add Prefix" && "Adds \"Status:\" before text"}
+                                {name === "Capitalize Words" && "Makes first letter of each word uppercase"}
+                                {name === "Custom Format" && "Start with a blank template"}
+                                {name === "No styling" && "No styling"}
```

Comment:
Could add these to the template map then loop over them.  This would allow for type safety as well.